clients/java/src/main/java/io/lakefs/clients/api/ApiClient.java

```diff
@@ -131,7 +131,7 @@ private void init() {
         json = new JSON();
 
         // Set default User-Agent.
-        setUserAgent("OpenAPI-Generator/0.1.0-SNAPSHOT/java");
+        setUserAgent("lakefs-java-sdk/dev-f1d18ae9.with.local.changes");
```

Comment:
Should we check this in? IIUC this will now change with every build.

Reply:
You are right I'll keep just the VERSION value - we will have 'dev' and every publish we will have the lakefs version without the git sha.